### ISSUE TITLE
Allow multiple copy lines to super-impose sprites

### ISSUE THREAD

**Issue by m-ender** [ClementSparrow/Pattern-Script#38]
When defining a sprite via sprite copy/transformations, it would be useful to allow several sprites to be copied on top of each other and transformed individually.

**Comment by ClementSparrow** [comment]
yeah, I was considering doing that, too.

**Comment by david-pfx** [comment]
Common situation for puzzle pieces is symmetry. Boring to define a large piece with much duplication. So, define one half or quarter and use copy/rotation to fill in the rest.

**Comment by david-pfx** [comment]
This is non-trivial to define. For example, what would this mean?
```
    Lava:Anim5
    Red Orange #CF4020
    copy: Lava:PrevAnim5 shift:left
    copy: overlay rot:up:right 
```
What does the rot apply to?

**Comment by ClementSparrow** [comment]
Yes, there is some ambiguity in the syntax, but I think it makes more sense if the transformations apply only to the last bitmap provided (if needed, you can also add the same transformation after the transformations for the other bitmaps provided).

Also, I'd like to be able to provide a bitmap directly without copying it from another object. Something like:

    Lava:Anim5
    Red Orange #CF4020
    copy: Lava:PrevAnim5 shift:left
    +
    00100
    01000
    11111
    01000
    00100
    rot:up:right

**Comment by david-pfx** [comment]
That's turning into a whole sub-language. No problem, but needs careful specification.
